### PR TITLE
Update message on CARINA-1002

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-7864782b93/).

File: _reference/2016-04-29-error-codes.md [hunk 1]
```diff
@@ -41,3 +41,9 @@ You attempted to run a container that adds Linux capabilities. This is not allow
 `Dropping capabilities not allowed by policy`
 
 You attempted to run a container that drops Linux capabilities. This is not allowed in Carina due to security restrictions, see [Understanding how Carina uses Docker Swarm][docker-swarm-carina]. To resolve this error, do not use the `--cap-drop` flag.
+
+### CARINA-1005
+
+`Provided source mount does not exist: <invalid_directory>`
+
+You attempted to run a container that mounts a volume from a non-existent directory of the node using a volume flag of the form `--volume /node/dir:/container/dir` or `-v /node/dir:/container/dir`.
```
Comment (everett-toews) [inline _reference/2016-04-29-error-codes.md]: I know it should be obvious but please provide a "To resolve this error, ..." sentence at the end.

File: _reference/2016-04-29-error-codes.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ You attempted to run a container in privileged mode. This is not allowed in Cari
 
 ### CARINA-1002
 
-`Host mount path(s) not allowed by policy`
+`Host mount path(s) not allowed by policy: <blocked_mount [, blocked_mount...]>`
 
 You attempted to run a container that mounts a volume from a restricted directory of the node using a volume flag of the form `--volume /node/dir:/container/dir` or `-v /node/dir:/container/dir`. This is not allowed in Carina due to security restrictions, see [Understanding how Carina uses Docker Swarm][docker-swarm-carina]. To resolve this error, see the section on [Volumes]({{ site.baseurl }}/docs/concepts/docker-swarm-carina/#volumes).
```
Comment (everett-toews) [inline _reference/2016-04-29-error-codes.md]: Can you also change this to "...restricted directory on the node..." while you're there?

File: _reference/2016-04-29-error-codes.md [hunk 3]
```diff
@@ -41,3 +41,9 @@ You attempted to run a container that adds Linux capabilities. This is not allow
 `Dropping capabilities not allowed by policy`
 
 You attempted to run a container that drops Linux capabilities. This is not allowed in Carina due to security restrictions, see [Understanding how Carina uses Docker Swarm][docker-swarm-carina]. To resolve this error, do not use the `--cap-drop` flag.
+
+### CARINA-1005
+
+`Provided source mount does not exist: <invalid_directory>`
+
+You attempted to run a container that mounts a volume from a non-existent directory of the node using a volume flag of the form `--volume /node/dir:/container/dir` or `-v /node/dir:/container/dir`. To resolve this error, provide a valid source directory for the mount.
```
Comment (everett-toews) [inline _reference/2016-04-29-error-codes.md]: Can you also change this to "...restricted directory on the node..." while you're there?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-da11bee04e/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-da11bee04e/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-2988b5c227/)

File: _reference/2016-04-29-error-codes.md [hunk 4]
```diff
@@ -41,3 +41,9 @@ You attempted to run a container that adds Linux capabilities. This is not allow
 `Dropping capabilities not allowed by policy`
 
 You attempted to run a container that drops Linux capabilities. This is not allowed in Carina due to security restrictions, see [Understanding how Carina uses Docker Swarm][docker-swarm-carina]. To resolve this error, do not use the `--cap-drop` flag.
+
+### CARINA-1005
+
+`Provided source mount does not exist: <invalid_directory>`
```
Comment (everett-toews) [inline _reference/2016-04-29-error-codes.md]: In CARINA-1002 you call them "host mount path(s)". Would it be better to use that same verbiage here?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-75994c483b/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-93d6f7b96d/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-14565bc01d/)

**Comment by everett-toews** [comment]
👍 

@lblackstone Merge whenever you think is appropriate.

**Comment by lblackstone** [comment]
@everett-toews 👍 
I don't have permission on the repo to merge, but good whenever.

**Pull request merged**